src/test/java/at/ac/tuwien/kr/alpha/solver/OmigaBenchmarksTest.java

```diff
@@ -74,19 +77,19 @@ public void testLocstrat_400() throws IOException {
 		test("locstrat", "locstrat-400.txt");
 	}
 
-	@Test(timeout = 10000)
-	@Ignore("disabled to save resources during CI")
+	@Test(timeout = 15000)
 	public void testReach_1() throws IOException {
 		test("reach", "reach-1.txt");
 	}
 
-	@Test(timeout = 10000)
+	@Test(timeout = 15000)
 	@Ignore("disabled to save resources during CI")
 	public void testReach_4() throws IOException {
 		test("reach", "reach-4.txt");
 	}
 
 	private void test(String folder, String aspFileName) throws IOException {
+		ignoreTestForNaiveSolver();
```

Comment:
Please move the conditional-ignore to the level of the unit test and not nested inside some (private) testing code, i.e., make the call to `ignoreTestForNaiveSolver();` at the beginning of `testReach_1()` and all other tests that should not be run with the naive solver. Actually, since these are the Omiga benchmarks, the naive solver should be able to run them in somewhat acceptable time. Please check what tests need to be ignored and which work fine.